modules/oeFax/controller/FaxServer.php

```diff
@@ -0,0 +1,125 @@
+<?php
+/**
+ * Fax Server SMS Module Member
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    Jerry Padgett <sjpadgett@gmail.com>
+ * @copyright Copyright (c) 2019 Jerry Padgett <sjpadgett@gmail.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+$ignoreAuth = 1;
+require_once(__DIR__ . "/../../../interface/globals.php");
+
+use OpenEMR\Common\Crypto\CryptoGen;
+
+class FaxServer
+{
+    private $baseDir;
+    private $crypto;
+    private $authToken;
+
+    public function __construct()
+    {
+        $this->baseDir = $GLOBALS['OE_SITE_DIR'] . DIRECTORY_SEPARATOR . "messageStore";
```

Comment:
TODO for me. figure out what messageStore directory is for.

Reply:
It's for rare downloaded pdfs and messages. I had bigger plans for it but haven't needed much yet. still need though..